src/vscode-dts/vscode.d.ts

```diff
@@ -10633,6 +10645,16 @@ declare module 'vscode' {
 		 */
 		readonly onDidChangeSelection: Event<TreeViewSelectionChangeEvent<T>>;
 
+		/**
+		 * Currently focused element.
+		 */
+		readonly focus: T | undefined;
```

Comment:
We try to avoid references to "focus" in our API and use "active" instead. If both an editor and a tree item can't both have focus, but both can have an active item.

Reply:
I can see how `focus` can be confusing. I changed it to `activeItem` and renamed subsequently. However, internally it is still referred to as "focus" (like `setFocus`, `focusedElement`, `focusedHandle`). Should those also be renamed or do we only care about the public API?

Reply:
We only care about the public API.